modules/oadp-kopia-configuring-algorithms.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-kopia-configuring-algorithms_{context}"]
+= Configuring the DPA to override Kopia hashing, encryption, and splitter algorithms
+
+{oadp-short} provides you with the option to override the default Kopia algorithms for hashing, encryption, and splitter to improve Kopia performance or to compare performance metrics. You can set the following environment variables in the `spec.configuration.velero.podConfig.env` section of the DPA:
```

Comment:
"provides you" is a form of anthropomorphism. How about:

> You can use an OADP option to...

![Screenshot from 2024-09-02 16-32-33](https://github.com/user-attachments/assets/f057cb0b-2c40-487d-86ad-9d301f89f97a)

Reply:
Done.